docs/proposals/03-loadbalancer.md

```diff
@@ -96,7 +96,7 @@ status:
 ```
 
 ```yaml
-apiVersion: networking.onmetal.de/v1alpha1
+apiVersion: networking.api.onmetal.de/v1alpha1
 kind: LoadBalancerRouting
```

Comment:
Which object should be reconciled **LoadBalancer** or **LoadBalancerRouting**?

Reply:
The `onmetal-api` controllers will apply `LoadBalancerRouting` from `LoadBalancer`.

Reply:
hope i clarified the text, as @adracus did here